tools/c7n_org/c7n_org/cli.py

```diff
@@ -458,6 +458,599 @@ def report(config, output, use, output_dir, accounts,
     writer.writerows(rows)
 
 
+def validate_basic(custodian_config, policy_file, fmt, check_mode, verbose):
+    """Run basic account-agnostic validation (Phase 1 logic).
+
+    Args:
+        custodian_config: Parsed policy configuration
+        policy_file: Path to policy file (for error reporting)
+        fmt: File format ('yml', 'yaml', 'json')
+        check_mode: Deprecation check mode (deprecated.SKIP, deprecated.STRICT, or None)
+        verbose: Verbose logging flag
+
+    Returns:
+        bool: True if validation passes, False otherwise
+    """
+    from c7n import schema, deprecated
```

Comment:
Are we late importing here to prevent circular imports or similar? If not, can these be moved to top-level imports?